app/models/kudo.rb

```diff
@@ -1,6 +1,6 @@
 class Kudo < ApplicationRecord
```

Comment:
it would be good to add assosiations between models `Kudo` and `CompanyValue`, than everything will be easier thanks to "Rails Magic"